tests/integration/plugins/test_rust.py

```diff
@@ -127,80 +74,18 @@ def test_rust_plugin_features(new_dir):
     assert output == "hello world\n"
 
 
-def test_rust_plugin_workspace(new_dir):
+def test_rust_plugin_workspace(new_dir, datadir):
     parts_yaml = textwrap.dedent(
         """\
         parts:
           foo:
             plugin: rust
-            source: .
+            source: test_rust/workspace
             rust-path: ["hello"]
         """
     )
     parts = yaml.safe_load(parts_yaml)
 
-    Path("Cargo.toml").write_text(
```

Comment:
Did these tests create stuff in the user's working directory? Ugly.

Reply:
no, new_dir changes cwd to a tmp directory